assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm

```diff
@@ -103,13 +103,16 @@ $(document).ready(function(){
 			## TEXT_AND_ATTACHMENT_ASSIGNMENT_SUBMISSION = 3;
 			## NON_ELECTRONIC_ASSIGNMENT_SUBMISSION = 4;
 			## SINGLE_ATTACHMENT_SUBMISSION = 5
+			## VIDEO_SUBMISSION = 7
 			<p class="instruction">
 				#if (($submissionType==1) || ($submissionType==3))
 					 $tlang.getString("stuviewsubm.asshasnot")
 				#elseif (($submissionType==2) || ($submissionType==5))
 				 	$tlang.getString("stuviewsubm.assattonly")
-				#elseif ($submissionType==5)
+				#elseif ($submissionType==6)
```

Comment:
I'd have spaces around the ==. You've done that elsewhere.

Reply:
That happens to me when I fix bugs!